src/frontend/src/lib/components/ui/AccessMethod.svelte

```diff
@@ -60,7 +63,7 @@
     <!-- Passkey -->
     <div
       class={[
-        "text-text-primary foldable-subgrid text-sm font-semibold nth-[2]:hidden",
+        `${isDisabled ? "text-text-disabled" : "text-text-primary"} foldable-subgrid text-sm font-semibold nth-[2]:hidden`,
```

Comment:
Using strings like this breaks IDE hint, also it's not needed since svelte uses clsx.
```suggestion
        "foldable-subgrid text-sm font-semibold nth-[2]:hidden",
        isDisabled ? "text-text-disabled" : "text-text-primary",
```